### DISCUSSION TITLE
Scan qr code

### DISCUSSION THREAD

**Discussion by lutungp** [kirimwa-id/forum#112] [Tanya Jawab] [answered]
![Screenshot 2022-04-09 152951](https://user-images.githubusercontent.com/25378929/162563550-2e74a2fd-7940-44aa-8ad5-0b03a6208883.png)
![Screenshot 2022-04-09 153019](https://user-images.githubusercontent.com/25378929/162563554-1f3b94ab-7598-4159-a9de-189d57e33086.png)
Saya mencoba kirimwa dengan laravel php, tetapi selalu mendapat response "Device is not authenticated, please rescan QR code".
Saya sudah mencoba scan qr code dari response api ini "https://api.kirimwa.id/v1/qr?device_id=862869040649282". dan device id tersebut juga sudah saya registerasi kan.

Untuk device id udah saya coba pakai imei maupun random id.

Tolong dibantu mas .

Thanks.

**Comment by rioastamal** [comment, marked as answer]
@lutungp Pastikan:

- WA anda bukan versi yang multi-device
- QR Code berlaku selama kurang lebih 15 detik jadi segera lalukan scan
